src/components/ValidatedInput/ValidSelectAsync.tsx

```diff
@@ -34,12 +42,19 @@ export const ValidSelectAsync = <OptionType, Group, Additional, isMulti extends
   className,
   dataQa,
   helperText,
+  isShowReloadWhileFetchFailed,
+  reloadMessage,
+  onReload,
+  reloaderStatus,
   ...props
 }: ValidSelectAsyncProps<OptionType, Group, Additional, isMulti>) => (
   <div className={classNames("valid-select-input", "valid-select-async-input", className)} data-qa-id={dataQa}>
     {!!label && <FieldLabel label={label} optionalLabel={optionalLabel} isRequired={isRequired} tooltip={tooltip} />}
     <AsyncPaginate {...props} className="styled-atlaskit-select" classNamePrefix={"styled-atlaskit-select"} />
     {isInvalid && <FieldErrorMessage message={errMessage} />}
+    {isShowReloadWhileFetchFailed && (
```

Comment:
do we really need this for valid select async? i don't see any place in which we set this prop to ValidSelectAsync